addon/components/link-to-component.js

```diff
@@ -15,6 +16,11 @@ export default LinkComponent.extend({
 
     let owner = getOwner(this);
 
+    assert(
+      `You attempted to use a link-to within a routeless engine, this is not supported.`,
```

Comment:
Ok, I've not tried it yet as I'm on mobile, but we may be able to lookup the config using the code below, then access the `modulePrefix` from there. 

```js
const config = owner.lookup('config:environment');
```

If we can use this to access the module prefix, I think it could be a good idea to scope the link-to's as well as add the assertion. This way we're effectively rendering the link-to component useless with the scope of a routeless engine.

Would you agree?